Description: Do not run tests in ./tests
 The toplevel tests are for the namespaced version of oslo.utils, so they
 are failing. This patch fixes the .testr.conf to not use them, as advised
 by Doug Hellman.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Updated: 2015-02-16

--- python-oslo.utils-1.2.1.orig/.testr.conf
+++ python-oslo.utils-1.2.1/.testr.conf
@@ -2,6 +2,6 @@
 test_command=OS_STDOUT_CAPTURE=${OS_STDOUT_CAPTURE:-1} \
              OS_STDERR_CAPTURE=${OS_STDERR_CAPTURE:-1} \
              OS_TEST_TIMEOUT=${OS_TEST_TIMEOUT:-60} \
-             ${PYTHON:-python} -m subunit.run discover -t ./ . $LISTOPT $IDOPTION
+             ${PYTHON:-python} -m subunit.run discover -t ./ ./oslo_utils $LISTOPT $IDOPTION
 test_id_option=--load-list $IDFILE
 test_list_option=--list
